server/datastore/mysql/apple_mdm.go

```diff
@@ -3513,3 +3513,21 @@ WHERE
 
 	return &res, nil
 }
+
+func (ds *Datastore) InsertMDMAppleDDMRequest(ctx context.Context, hostUUID, message_type, raw_json string) error {
```

Comment:
this is annoying because the db tables use snake_case, but:

```suggestion
func (ds *Datastore) InsertMDMAppleDDMRequest(ctx context.Context, hostUUID, messageType, rawJSON string) error {
```

Reply:
Oh I fixed this while talking with you yesterday and forgot to commit the changes lol